### ISSUE TITLE
Accessibility/Client Scopes/Scope Tab: Moderate Violation

### ISSUE THREAD

**Issue by agagancarczyk** [keycloak/keycloak#19445]
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the latest release

### Area

admin/ui

### Describe the bug

![Screenshot 2023-03-30 at 14 32 25](https://user-images.githubusercontent.com/4890675/228852937-ed48c6ad-03b0-4507-9f6b-87128b84c9f5.png)

### Version

21.0.0

### Expected behavior

Violation is fixed

### Actual behavior

Violation is not fixed

### How to Reproduce?

Use Deque browser tool
